cmake/sample_defs/sample_mission_cfg.h

```diff
@@ -35,6 +35,11 @@
 #ifndef SAMPLE_MISSION_CFG_H
 #define SAMPLE_MISSION_CFG_H
 
+/*
+** Includes
+*/
+#include "cfe_es_api_typedefs.h"
+
```

Comment:
Even though this technically does reference a "CFE_ES_CrcType" enum label now, we don't strictly need this inclusion here, because its just a preprocessor macro, and thus only gets evaluated when it used/referenced.  Anyone who uses the DEFAULT_CRC macro would, by definition, have to include the ES API header to get the prototype for the actual call.  

As a "config file" - I think the preference is to keep these config headers (mission/platform) to be very simple, because they can be used in tools and things like `elf2cfetbl` that might not have access to the ES API header, but can potentially use some of the other values in here.

So while an API or type-defining header should be fully self-contained (i.e. include all its dependencies) that is not really necessary for a pure config header.

There is precedent for this; some of the values in here are true/false which technically depend on `stdbool.h` and also there are examples in the platform config of referencing enum values that are not directly included in the config file.

Reply:
> Getting there... but we need to revert back the rest of the `CFE_MISSION_ES_DEFAULT_CRC` references.

Added them all back - changes are only to the direct references to the enum values.